### PR TITLE
Minor edits to MSE for Audio Update

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: src/content/en/updates/2015/06/Media-Source-Extensions-for-Audio.md [hunk 1]
```diff
@@ -352,18 +352,18 @@ Now that we have the total number of samples we can move on to reading out the n
...
+With that we have a complete function for parsing the vast majority of gapless content. Edge cases certainly abound though, so caution is recommended before using similar code in production.
 
 ## Appendix C: On Garbage Collection
 
-Memory belonging to `SourceBuffer`s is actively [garbage collected](https://en.wikipedia.org/wiki/Garbage_collection_(computer_science)) according to content type, platform specific limits, and the current play position. In Chrome, memory will first be reclaimed from already played buffers. However, if memory usage exceeds platform specific limits, it will remove memory from unplayed buffers.
+Memory belonging to `SourceBuffer` instances s is actively [garbage collected](https://en.wikipedia.org/wiki/Garbage_collection_(computer_science)) according to content type, platform specific limits, and the current play position. In Chrome, memory will first be reclaimed from already played buffers. However, if memory usage exceeds platform specific limits, it will remove memory from unplayed buffers.
```
Comment (beaufortfrancois) [inline src/content/en/updates/2015/06/Media-Source-Extensions-for-Audio.md]: Remove trailing `s`

**Comment by beaufortfrancois** [comment]
We might also want to use TextDecoder in `Appendix B: Parsing Gapless Metadata`:

```diff
function ParseGaplessData(arrayBuffer) {
  // Gapless data is generally within the first 512 bytes, so limit parsing.
-  var byteStr = String.fromCharCode.apply(
-      null, new Uint8Array(arrayBuffer.slice(0, 512)));
+ var byteStr = new TextDecoder().decode(arrayBuffer.slice(0, 512));
```

**Comment by beaufortfrancois** [comment]
In "Basic Setup", can we remove the option `false` when adding event listener?

```diff
mediaSource.addEventListener('sourceopen', function() {
  var sourceBuffer = mediaSource.addSourceBuffer('audio/mpeg');

  function onAudioLoaded(data, index) {
    // Append the ArrayBuffer data into our new SourceBuffer.
    sourceBuffer.appendBuffer(data);
  }

  // Retrieve an audio segment via XHR.  For simplicity, we're retrieving the
  // entire segment at once, but we could also retrieve it in chunks and append
  // each chunk separately.  MSE will take care of assembling the pieces.
  GET('sintel/sintel_0.mp3', function(data) { onAudioLoaded(data, 0); } );
-}, false);
+});

**Comment by beaufortfrancois** [comment]
In "Basic Setup", let's drop the `window` before `URL`.

```diff
-audio.src = window.URL.createObjectURL(mediaSource);
+audio.src = URL.createObjectURL(mediaSource);

**Comment by beaufortfrancois** [comment]
In "Example Code", shall we also revoke URL once `endOfStream` is called?

```diff
        // We've loaded all available segments, so tell MediaSource there are no
        // more buffers which will be appended.
        mediaSource.endOfStream();
+       URL.revokeObjectURL(audio.src);

**Comment by beaufortfrancois** [comment]
We should also update the poster images because they don't always reflect the appropriate video.
I suggest : 
- either remove poster attributes from video
- or create another poster for green lines video.

**Comment by beaufortfrancois** [comment]
We should also reflect our changes to https://github.com/samdutton/simpl/tree/gh-pages/mse/audio.

**Comment by jpmedley** [comment]
@beaufortfrancois PTAL

**Comment by beaufortfrancois** [comment]
@jpmedley 👍

**Pull request merged**